actor/src/main/scala-2.12/org/apache/pekko/util/ByteString.scala

```diff
@@ -566,6 +572,9 @@ object ByteString {
 
     def asByteBuffers: scala.collection.immutable.Iterable[ByteBuffer] = bytestrings.map { _.asByteBuffer }
 
+    override def asInputStream: InputStream =
+      new SequenceInputStream(bytestrings.map(_.asInputStream).iterator.asJavaEnumeration)
```

Comment:
```suggestion
      new SequenceInputStream(bytestrings.iterator.map(_.asInputStream).asJavaEnumeration)
```

Reply:
Feel free, I have back hometown

Reply:
But still, it's worth optimizing even income is insignificant！

Reply:
turns out that @He-Pin 's suggestion does seem to run a little faster - so I've made the change